jobs/scripts/4.3-tests.sh

```diff
@@ -1,9 +1,28 @@
 #!/bin/sh
 # run buildout and tests
-{pybinary} bootstrap.py -c jenkins.cfg
-bin/buildout buildout:git-clone-depth=1 -c jenkins.cfg
```

Comment:
Without ``jenkins.cfg`` the ``jenkins-alltests*`` parts are not generated, this is the script used for 4.3